content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
-
-Pick a [hosted solution](/docs/setup/pick-right-solution/#hosted-solutions).
-
-## Turnkey – Cloud Solutions
-
-
-These solutions allow you to create Kubernetes clusters with only a few commands and 
-are actively developed and have active community support. They can also be hosted on 
-a range of Cloud IaaS providers, but they offer more freedom and flexibility in 
-exchange for effort. 
-
-You should pick a turnkey cloud solution if you:
-
-* Want more control over your clusters than the hosted solutions allow
-* Want to take on more operations ownership 
-
-Pick a [turnkey cloud solution](/docs/setup/pick-right-solution/#turnkey-cloud-solutions)
-
-## Turnkey – On-Premises Solutions
-
-These solutions allow you to create Kubernetes clusters on your internal, secure,
-cloud network with only a few commands.
-
-You should pick a on-prem turnkey cloud solution if you:
-
-* Want to deploy clusters on your private cloud network
-* Have a dedicated SRE team
-* Have the resources to host and monitor your clusters
-
-Pick an [on-prem turnkey cloud solution](/docs/setup/pick-right-solution/#on-premises-turnkey-cloud-solutions).
-
-## Custom Solutions
-
-Custom solutions give you the most freedom over your clusters but require the 
-most expertise. These solutions range from bare-metal to cloud providers on 
-different operating systems.
-
-Pick a [custom solution](/docs/setup/pick-right-solution/#custom-solutions).
+## Learning environment
+
+If you want to get familiar with Kubernetes, you can use the Docker-based solutions. You can use tools that are supported by the Kubernetes community or the tools in the ecosystem to setup a Kubernetes cluster on a local-machine. 
+
+{{< table caption="Local-machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ecosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
+| [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) | [Minishift](https://docs.okd.io/latest/minishift/)|
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
+|                     | [k3s](https://k3s.io)|
+|                     | [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/)|
+
+
+## Production environment
+
+The production environment solutions diagram lists the possible abstractions of a Kubernetes cluster and the level of expertise involved in managing the operations of a cluster.
```

Comment:
Either say "The ... diagram below ..." or "This ... diagram ..."

Reply:
@Rajakavitha1, Personally, I think the image of the table is two big for the page and the colors are too strong.
The other tables use text that is very small. I think a Markdown table would suffice to convey the information.  Do you have any links to other info (dev ops, cluster admins, architects)  to corroborate the abstractions?

Reply:
@kbhawkey Yes, I agree....I would love to explore color options in markdown table bring out the distinction. 
This is a WIP PR and I was insprired by: https://twitter.com/wswmuc/status/681217518778871810 and I created https://github.com/Rajakavitha1/Images/blob/master/KubernetesSolutions.svg.

- Managed by Providers and Self Managed was based on my understanding of GKE vs GCE vs GKE On-Prem | Google Cloud.
  References:
- https://medium.com/google-cloud/gcp-the-google-cloud-platform-compute-stack-explained-c4ebdccd299b
- https://www.slideshare.net/peihsinsu/google-cloud-computing-compares-gce-gae-and-gke
- https://aws.amazon.com/blogs/architecture/compute-abstractions-on-aws-a-visual-story/
- https://www.weave.works/technologies/weaveworks-on-aws/

That is why i was seeking technical review from @steveperry-53.